### DISCUSSION TITLE
scrollTo exact Top and Left location

### DISCUSSION THREAD

**Discussion by babakansari** [glideapps/glide-data-grid#466] [Q&A] [answered]
Hi, We are trying to synchronize scrolling multiple Grids on a single page. The problem we are facing is that the `scrollTo` method does not have a way to move to an exact (Top, Left) location (cell). It seems intentionally doesn't scroll when the cell is already. We want to scrollTo cell to be top first cell.

Is it possible to expose that functionality? What is the other alternative? Any suggestion?

Thanks!

**Comment by jassmith** [comment, marked as answer]
I am in the middle of overhauling this functionality for another consumer as well. I will make sure this can be done as well as pixel perfect adjustments.

**Reply by babakansari** [reply]
Is there any timeframe to release this feature? Thanks!

**Reply by jassmith** [reply]
I will be starting to push betas in the next couple days. You will be able to use those.

**Reply by babakansari** [reply]
Thanks, @jassmith. That is great news!

**Comment by jassmith** [comment]
Can you please file an issue and tag it feature request

**Reply by babakansari** [reply]
Hello, Thanks for the update. I created the following issue:

https://github.com/glideapps/glide-data-grid/issues/468

**Reply by jassmith** [reply]
this is available in an alpha for a bit now, check the 5.1.0 series.

**Comment by sudhansu36** [comment]
We Implemented this feature of synchronize scrolling in our app and we facing following problem: 
There is a problem in synchronized scrolling as we move left to right or vice versa. Some times it doesn't work properly as mentioned in the screenshot. Is it a performance issue? Could you please help us with any solutions?
@jassmith @babakansari 

https://user-images.githubusercontent.com/49445013/210718506-c1eb5284-d5e5-42df-95e8-fa91a445f65d.png
